docs/manual/java/releases/Migration16.md

```diff
@@ -62,3 +62,17 @@ akka.serialization.jackson {
 #### JSON Compression threshold
 
 When marking a serializable class with `CompressedJsonable` compression will only kick in when the serialized representation goes past a threshold. The default value for `akka.serialization.jackson.jackson-json-gzip.compress-larger-than` is 32 Kilobytes. As mentioned above, this setting was previously configure by `lagom.serialization.json.compress-larger-than` and defaulted to 1024 bytes. (See [#1983](https://github.com/lagom/lagom/pull/1983))
+
+## Lagom Cluster
+
+Lagom 1.6.0 builds on Akka 2.6.0 that uses a new Akka Remote implementation called Artery. Artery is enabled by default in Lagom and replaces the previous Akka Remote protocol (aka. Akka Remote Classic). If you are using Lagom in a clustered setup, you will need to shutdown all nodes before updating, unless you choose to not use Artery. Read on to "Upgrading a production system" section to learn how to opt-out from Artery.
+
+## Upgrading a production system
```

Comment:
The text here is already preparing the ground for other 'opt-out' options, like `ddata`.

Reply:
Oh, I see that we have already some work in progress here https://github.com/lagom/lagom/pull/2007

We solve the merge conflicts when needed